lib/Geometrics/Math/CMakeLists.txt

```diff
@@ -1,4 +1,3 @@
-target_sources(${LIBRARY_NAME} PUBLIC
-    BartaMathLibrary.cpp
-    QuadraticEquation.cpp
-)
\ No newline at end of file
+target_sources(
+    ${LIBRARY_NAME} PUBLIC QuadraticEquation.cpp
```

Comment:
I think we should move `QuadraticEquation` files to the `Utilities` directory as well.